tests/queries/0_stateless/02883_array_scalar_mult_div_modulo.sql

```diff
@@ -0,0 +1,18 @@
+SELECT materialize([2, 3, 5]) * materialize(7);
+SELECT [2, 3, 5] * materialize(7);
```

Comment:
`materialize` is a good function, but this test missed a bug.
Try something like `select [2, 3, 5] * number from numbers(5)`, I think it should return a wrong result now.

Also try something like `select range(number) * 42 from numbers(5)`.